gns3/main_window.py

```diff
@@ -376,6 +378,9 @@ def _newProjectActionSlot(self):
             else:
                 self._createTemporaryProject()
 
+    def _IOUVMConverterActionSlot(self):
+        subprocess.Popen([shutil.which("gns3-iouvm-converter")])
```

Comment:
shutil.which returns None if it cannot find a command.

https://app.getsentry.com/gns3/gns3-gui/group/73471545/